src/views/ResLife/components/RAView/components/RD-OnCall/index.jsx

```diff
@@ -14,12 +14,14 @@ import {
 import { styled } from '@mui/material/styles';
 import ScottieMascot from 'views/ResLife/ScottieMascot.png';
 import { getRDOnCall } from 'services/residentLife/RD_OnCall';
+import { formatPhoneNumber } from 'views/ResLife/utils/formatPhoneNumber/formatPhoneNumber';
```

Comment:
If this is a reusable library function, you should move it to a better location for reuse, such as somewhere in `src/services/residentLife`